### PR TITLE
Fix release build and make discovery period configurable

### PR DESCRIPTION
**Pull request by AndrejMitrovic**
Let's see if CI likes the first commit.

### CODE REVIEW

**Review by linked0 — APPROVED** [review]
LGTM

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by kchulj — APPROVED** [review]
LGTM

File: source/agora/node/Ledger.d [hunk 1]
```diff
@@ -868,8 +868,8 @@ public class Ledger
             if (CR == Point.init)
                 return "Block: Couldn't find commitment for this validator";
             Point R = CR + challenge.toPoint();
-            sum_K = sum_K == Point.init ? K : (sum_K + K);
-            sum_R = sum_R == Point.init ? R : (sum_R + R);
+            sum_K = sum_K + K;
+            sum_R = sum_R + R;
```
Comment (Geod24) [inline source/agora/node/Ledger.d:872]: I didn't know we fixed this in `crypto`, nice!

**Comment by Geod24** [comment]
This times out on `ValidatorCleanRestart`.

File: source/agora/node/Validator.d [hunk 2]
```diff
@@ -245,13 +245,9 @@ public class Validator : FullNode, API
...
-        this.taskman.runTask(
-        ()
-        {
-            void discover () { this.network.discover(this.required_peer_keys); }
-            discover();  // avoid delay
-            this.timers ~= this.taskman.setTimer(5.seconds, &discover, Periodic.Yes);
-        });
+        void discover () { this.network.discover(this.required_peer_keys); }
+        discover();  // avoid delay
+        this.timers ~= this.taskman.setTimer(5.seconds, &discover, Periodic.Yes);
```
Comment (Geod24) [inline source/agora/node/Validator.d]: This interval should really be configurable

Comment (AndrejMitrovic) [reply source/agora/node/Validator.d]: Good point, let me do it now.

**Comment by Geod24** [comment]
Look at the integration test logs. First commit doesn't seem to do good there.

**Comment by AndrejMitrovic** [comment]
Ah so it was there because of vibe.d.. then I'll have to nuke the commit.

**Review by Geod24 — APPROVED** [review]
Can you rebase on v0.x.x ?

**Comment by AndrejMitrovic** [comment]
Rebased.

**Comment by Geod24** [comment]
> Rebased.

CI disagrees

**Pull request merged**